pkg/sql/executor_statement_metrics.go

```diff
@@ -205,18 +205,17 @@ func (ex *connExecutor) recordStatementSummary(
 
 	stmtFingerprintID, err :=
 		ex.statsCollector.RecordStatement(ctx, recordedStmtStatsKey, recordedStmtStats)
-
-	// TODO(xinhaoz): This can be set directly within statsCollector once
-	// https://github.com/cockroachdb/cockroach/pull/123698 is merged.
-	ex.statsCollector.SetStatementFingerprintID(stmtFingerprintID)
-
 	if err != nil {
 		if log.V(1) {
 			log.Warningf(ctx, "failed to record statement: %s", err)
 		}
 		ex.server.ServerMetrics.StatsMetrics.DiscardedStatsCount.Inc(1)
 	}
 
+	// TODO(xinhaoz): This can be set directly within statsCollector once
```

Comment:
Was this supposed to move?

Reply:
Yeah I just moved the err checking block to be right after the function call - I think the diff just looks weird bc it makes it seem like I moved this block instead.